src/main/scala-2.11/markets/actors/participants/OrderIssuer.scala

```diff
@@ -15,41 +15,50 @@ limitations under the License.
 */
 package markets.actors.participants
 
-import markets.orders.Order
+
+import markets.actors.participants.strategies.OrderIssuingStrategy
+import markets.orders.{AskOrder, BidOrder}
 import markets.orders.limit.{LimitAskOrder, LimitBidOrder}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2585203069/issues/source?bid=2921361&fileBranchId=3304063#l21)